_posts/theme/2021-08-24-Dracula.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: theme
+title: Dracula
+category: theme
+homepage: https://github.com/Teyler7/dracula-typora-theme
+download: https://github.com/Teyler7/dracula-typora-theme/archive/refs/heads/main.zip
+built-in: false
+author: Teyler7
+thumbnail: dracula-typora.png
```

Comment:
This file seems not exist

Reply:
My apologies. Thank you for calling this out. I have updated my file path for my thumbnail image.